playbooks/ceph.yml

```diff
@@ -282,8 +284,8 @@
         cifmw_cephadm_dashboard_crt: "{{ cifmw_cephadm_certificate }}"
         cifmw_cephadm_dashboard_key: "{{ cifmw_cephadm_key }}"
 
-
     - name: Create cephfs volume
+      when: cifmw_ceph_daemons_layout.cephfs_enabled  | default(true) | bool
```

Comment:
I'm sorry to go back here, but I think we should move all these defaults to `false` (as John suggested at the beginning). We deploy rbd only unless something else is specified via overrides.
@fultonj @katarimanojk is that a sane approach?